src/index.md

```diff
@@ -28,12 +28,12 @@ configurationTabs:
     value: config_A
     visualSrc: /assets/config-a.svg
     mobileVisualSrc: /assets/config-a-mobile.svg
-    textContent: Before Stratum V2, transaction sets were selected by pools. With this SV2 configuration they’re selected by individual miners, making the network more censorship-resistant. Miners run SV2 firmware, connecting to the SV2 proxy server. Miners can pick their transactions locally and declare them to an SV2 pool.
+    textContent: Miners can utilize Stratum v2 without updating to SV2 firmware. Connect your SV1 firmware devices through Translation Proxy which facilitates the conversion of SV1 messages to SV2 for communication with an SV2 pool. Miners handle transaction selection locally via their own Template Provider, and declare them to a pool. If a pool rejects the proposed transactions, miners automatically switch to an alternative pool. Should all configured pools decline the proposal, miners fallback to solo mining.
```

Comment:
Do we want to use Sv2/v1 instead of SV2/V1 everywhere?

Reply:
Yeah Stratum v2 so should be  Sv2 if that's what you mean?

Reply:
> Yeah Stratum v2 so should be Sv2 if that's what you mean?

Yeah exactly 👍

Reply:
Based on recent discussions decided to uppercase everything for now both Stratum V2 and SV2, will update based on discussion in #193